identifiers/country-za/README.md

```diff
@@ -0,0 +1,6 @@
+# OCD Division Identifier
+
+This folder lists the following administrative divisions of South Africa: 
+* The provinces of South Africa ([ref](https://en.wikipedia.org/wiki/Administrative_divisions_of_South_Africa)) as **province**.
+
+The OCD identifiers are created based on the ISO 3166-2 code (Used [source](https://en.wikipedia.org/wiki/Provinces_of_South_Africa))
```

Comment:
Just a nitpick but https://en.wikipedia.org/wiki/ISO_3166-2:ZA is probably a better URL source to reference for the ISO 3166-2 codes.

Reply:
Goog point. Done.